htdocs/compta/paiement/card.php

```diff
@@ -310,6 +312,33 @@
 // Amount
 print '<tr><td>'.$langs->trans('Amount').'</td><td>'.price($object->amount,'',$langs,0,-1,-1,$conf->currency).'</td></tr>';
 
+// External link
+if (!empty($object->payment_id) && !empty($object->payment_site) && $object->payment_site == 'stripe') {
```

Comment:
I think we should store into payment_site the value 'stripe' or 'stripetest' because we can have in same table some record done with test mode and others done with real mode.
What do you think of changing first into the creation of payment 2 different values ?